genrl/deep/bandit/data_bandits/mushroom_bandit.py

```diff
@@ -0,0 +1,144 @@
+from pathlib import Path
+from typing import Tuple, Union
+
+import numpy as np
+import pandas as pd
+import torch
+
+from .data_bandit import DataBasedBandit, download_data
```

Comment:
We're moving away from relative paths. So you'd have to change this at a lot of places now.

Reply:
Yeah ok, will change those